### PR TITLE
15 setup api endpoints

### PR DESCRIPTION
**Pull request by fhnaseer**

### CODE REVIEW

File: api/helpers/modelValidator.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    if (isLineChart(data))
+        return environment.scatterChartString;
+    if (isPieChart(data))
+        return environment.pieChartString;
+    if (isScatterChart(data))
+        return environment.scatterChartString;
+    return null;
+}
+
+function isHotogram(data) {
```
Comment (syxolk) [inline api/helpers/modelValidator.js]: Here's a spelling mistake

File: api/helpers/modelValidator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+var environment = require('./environment');
+
+exports.getChartType = function (data) {
+    if (isHotogram(data))
```
Comment (syxolk) [inline api/helpers/modelValidator.js]: Spelling mistake

**Review by syxolk — CHANGES_REQUESTED** [review]
Just one spelling mistake

**Comment by syxolk** [comment]
Travis seems to fail because it cannot find `.travis.yml`.

**Comment by fhnaseer** [comment]
i merged travis changes,

**Review by syxolk — APPROVED** [review]

**Pull request merged**